docs/04-operation-guides/troubleshooting/smgt-01-hb.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Helm Broker troubleshooting
+---
+
+## Possible `FAILED` status for created ServiceInstances
+
+If your ServiceInstance creation was successful and yet the release is marked as `FAILED` on the releases list when running the `helm list` command, it means that there is an error on the Helm's side that was not passed on to the Helm Broker. To get the error details, check the Helm release status.
```

Comment:
```suggestion
## Cause
There is an error on the Helm's side that was not passed on to the Helm Broker.

## Remedy
To get the error details, check the Helm release status.
```

Learning the error details doesn't fix them - how can users find out how to get rid of the FAILED status?